integration-tests/gatsby-source-wordpress/__tests__/index.js

```diff
@@ -60,6 +60,10 @@ describe(`[gatsby-source-wordpress] Run tests on develop build`, () => {
   let gatsbyDevelopProcess
 
   beforeAll(async () => {
+    if (process.env.SKIP_BEFORE_ALL) {
```

Comment:
It's used to develop the tests without needing to re-run gatsby over and over. Essentially you can start `gatsby develop` separately and then run the tests with this env var so the tests run in a second instead of 30 seconds